product-updates/knowledge-center/index.html

```diff
@@ -329,6 +333,61 @@ <h2 class="usa-card__heading">Use an existing Login.gov Account</h2>
 
         <h2>What's new in TDP</h2>
 
+        <h3 style="margin-top: 32px;">October 10th 2023 (v 3.1.6)</h3>
+        <h4 style="margin-top: 16px; color: #595959;">Added:</h4>
+        <ul class="usa-icon-list">
+          <li class="usa-icon-list__item" style="margin-top: 16px;">
+            <div class="usa-icon-list__icon text-ink">
+              <img src="../img/added.svg" aria-hidden="true" style="background-color: #d3ffcc; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+            </div>
+            <div class="usa-icon-list__content">
+              <h5 class="usa-icon-list__title">
+                Additions to knowledge center FAQs
+              </h5>
+              <p>
+                The knowledge center now offers new guidance including where to go for help if you encounter an unexpectedly high volume of errors in your data, how to proceed in a situation where you cannot correct a certain type of error, and how to utilize resubmission to improve your data quality and completeness throughout a fiscal quarter.
+              </p>
+            </div>
+          </li>
+        </ul>
+
+          <h4 style="margin-top: 16px; color: #595959;">Changed / Fixed:</h4>
+        <ul class="usa-icon-list">
+
+    <li class="usa-icon-list__item" style="margin-top: 16px;">
+      <div class="usa-icon-list__icon text-ink">
+          <img src="../img/changed.svg" aria-hidden="true" style="background-color: #b3e2ff; border-radius: 5px; width: 18px; height: 18px; position: relative; top: 2px; margin-right: 5px; margin-left: 5px;"/>
+      </div>
+      <div class="usa-icon-list__content">
+        <h5 class="usa-icon-list__title">
+          Increased actionability of file upload error messages
+        </h5>
+        <p>
+      In cases where a user attempts to upload an incompatible file type to TDP we're now triggering a more <a href="lightbox">descriptive error message</a> that describes which file types are accepted and compatible with TDP; Files ending in .txt, .ms##, .ts##, or .ts###. The filename of the incompatible file also now stays onscreen so that it can be more easily compared to the list of accepted file types. 
```

Comment:
```suggestion
      In cases where a user attempts to upload an incompatible file type to TDP we're now triggering a more <a href="lightbox">descriptive error message</a> that describes which file types are accepted and compatible with TDP; acceptable file types can end in: .txt, .ms##, .ts##, or .ts###. The filename of the incompatible file also now stays onscreen so that it can be more easily compared to the list of accepted file types. 
```